examples/integration-tutorials/exposing-several-services-as-a-single-service/reserve_appointment_service.bal

```diff
@@ -0,0 +1,83 @@
+import ballerina/http;
+import ballerina/log;
+
+http:Client hospitalEP = new("http://localhost:9090");
```

Comment:
We need to add variable and function comments

Reply:
done